app/templates/components/component-tree-item.hbs

```diff
@@ -1,106 +1,109 @@
 <div
   style={{@item.style}}
   class="
-    component-tree-item
+    component-tree-item relative flex items-center mx-1 rounded
+    {{if @item.hasInstance "cursor-pointer" "cursor-default"}}
     {{if @item.isComponent "component-tree-item--component"}}
-    {{if @item.hasInstance "component-tree-item--has-instance"}}
-    {{if @item.isSelected "component-tree-item--selected"}}
-    {{if @item.isHighlighted "component-tree-item--highlighted"}}"
```

Comment:
This was originally used to subtly highlight children of a selected component. At some point that changed and was being used for the hover state. There was a visible lag.

Reply:
Do we still highlight the children?

Reply:
Nope. Not sure if we should consider this a regression or pretend it never existed :-P

Reply:
When was the last time it worked? If it worked right before the Octane stuff, we should probably fix it.

Reply:
I think it got lost in the flurry of Octane changes.